tensorflow/core/kernels/string_split_op.cc

```diff
@@ -28,19 +28,62 @@ namespace tensorflow {
 
 namespace {
 
-std::vector<string> Split(const string& str, const string& delimiter,
-                          const bool skipEmpty) {
+Status Split(std::vector<string>& char_vector, const string& str,
```

Comment:
We don't do references. Instead, pass the output as the last parameter, as a pointer:

```Status Split(const string& str, const string& delimiter, ..., std::vector<string>* char_vector)
```

Reply:
Thanks @drpngx. Done.